microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -89,29 +89,35 @@ For full support for the latest APIs in a release version of the SDK, the Runtim
 
 To try out new forthcoming features that are in development, use experimental APIs.  Experimental APIs are contained in prerelease versions of the WebView2 SDK, but not in release versions of the WebView2 SDK.
 
+New APIs are introduced in phases as follows:
+
+1. Experimental in prerelease.
+1. Promoted to stable in prerelease.
+1. Promoted to stable in release; that is, promoted to release.
+
 
 ### Developing with experimental APIs and providing feedback
 
-The experimental APIs in a WebView2 SDK _prerelease_ package aren't guaranteed to be forward-compatible and might be removed in future Runtime updates.  When a _prerelease_ version of the WebView2 SDK is initially made available, that SDK only works with Microsoft Edge Canary.  Soon after that, the prerelease SDK also works with the Beta and Dev channels.  Use a prerelease SDK to try out new APIs early and provide feedback before the new APIs are promoted to become stable, forward-compatible APIs.
+The experimental APIs in a WebView2 SDK _prerelease_ package aren't guaranteed to be forward-compatible and might be removed in future Runtime updates.  When a _prerelease_ version of the WebView2 SDK is initially made available, that SDK only works with Microsoft Edge Canary.  Soon after that, the prerelease SDK also works with the Beta and Dev channels.  Use a prerelease SDK to try out new APIs early and provide feedback before the new APIs are promoted to become stable,<!-- todo: change "stable" to "prerelease stable" or to "release"? --> forward-compatible APIs.
```

Comment:
@liminzhu @novac42 -- see "todo" in this file.   Can we clarify the two levels of "stable": prerelease stable vs. release stable?  
Wording that uses "stable" should acknowledge there are two levels of "stable", and should address explicitly whether prerelease stable and/or release stable is meant.

Reply:
```suggestion
The experimental APIs in a WebView2 SDK _prerelease_ package aren't guaranteed to be forward-compatible and might be removed in future Runtime updates.  When a _prerelease_ version of the WebView2 SDK is initially made available, that SDK only works with Microsoft Edge Canary.  Soon after that, the prerelease SDK also works with the Beta and Dev channels.  Use a prerelease SDK to try out new APIs early and provide feedback before the new APIs are promoted to stable release.
```
wording suggestion per Lingyan